unit_tests/test_aggregator.py

```diff
@@ -294,84 +262,78 @@ def test_calculate_grid_fee__return(aggregator_explicit):
 
 
 class TestExecuteBatchCommands:
-    """Includes methods which test execute_batch_commands method."""
+    """Test the execute_batch_commands behaviors."""
 
     @staticmethod
-    def test_execute_batch_commands__command_buffer_length_zero(aggregator_explicit):
+    def test_execute_batch_commands_command_buffer_length_zero(aggregator_explicit):
         with patch("gsy_e_sdk.aggregator.ClientCommandBuffer.buffer_length",
                    new_callable=PropertyMock, return_value=0):
             assert aggregator_explicit.execute_batch_commands() is None
 
     @staticmethod
-    def test_execute_batch_commands__execute_batch_called(aggregator_explicit):
-        with patch("gsy_e_sdk.aggregator.ClientCommandBuffer.execute_batch") as mocked_method:
-            aggregator_explicit.start_websocket_connection()
-            aggregator_explicit.execute_batch_commands()
-            mocked_method.assert_called_once()
+    def test_execute_batch_commands_execute_batch_called(aggregator_explicit):
+        aggregator_explicit.start_websocket_connection()
+        aggregator_explicit.execute_batch_commands()
+        aggregator_explicit._client_command_buffer.execute_batch.assert_called_once()
 
     @staticmethod
-    def test_execute_batch_commands__validate_all_uuids_in_list(aggregator_explicit):
-        with patch("gsy_e_sdk.aggregator.Aggregator"
-                   + "._all_uuids_in_selected_device_uuid_list") as mocked_method:
-            aggregator_explicit.start_websocket_connection()
-            aggregator_explicit.execute_batch_commands()
-            mocked_method.assert_called()
+    def test_execute_batch_commands_validate_all_uuids_in_list(aggregator_explicit):
+        aggregator_explicit.start_websocket_connection()
+        aggregator_explicit.execute_batch_commands()
+        aggregator_explicit._all_uuids_in_selected_device_uuid_list.assert_called()
 
     @staticmethod
-    def test_execute_batch_commands__post_request_called_with_args(aggregator_explicit):
-        args_ = (f"{'/aggregator_prefix/'}batch-commands",
```

Comment:
Why do we need to append the _ at the end of the variable name?

Reply:
I did that because I thought it might conflict with a python reserved word. I understand from your suggestion it does not :P